presets/4.3/rates/mouseFreestyle.txt

```diff
@@ -4,9 +4,15 @@
 #$ STATUS: COMMUNITY
 #$ KEYWORDS: freestyle
 #$ AUTHOR: mouseFPV
+
 #$ PARSER: MARKED
-#$ DESCRIPTION: ### These are the freestyle rates used by [Youtube Freestyler](https://www.youtube.com/c/mouseFPV) and Tuner "mouseFPV"
-#$ DESCRIPTION:
+
+#$ DESCRIPTION: <br>
```

Comment:
Are you sure BR is needed here?

Reply:
It will include a little margin on top of the logo so that it is not too close to the top. Just a styling thing.

Reply:
> It will include a little margin on top of the logo so that it is not too close to the top. Just a styling thing. Maybe it also works without `<br>`. I will check.

they also say that any HTML black must be surrounded by empty lines:
https://daringfireball.net/projects/markdown/syntax#html

not sure if you also have to insert an empty line at the very beginning.

Reply:
I see. Seems like Marker doesn't render <br> tags. Do you know how to add line breaks?